debian/cuttlefish-common.init

```diff
@@ -114,7 +76,7 @@ destroy_interface() {
     tap="$(printf cvd-${1}tap-%02d $2)"
```

Comment:
consistency says $2 should be "${2}"
and "cvd-${1}tap-%02d"

(google shell style says always "${}" wrap var expansions if you can, although $1 is actually an exception to the ${} wrapping requirement, but not to "" wrapping)